spec/hydra/works/services/characterization_service_spec.rb

```diff
@@ -81,20 +81,25 @@
   end
 
   context "passing an object that does not have matching properties" do
-    let!(:current_schemas) { ActiveFedora::WithMetadata::DefaultMetadataClassFactory.file_metadata_schemas }
-
     let(:characterization) { class_double("Hydra::FileCharacterization").as_stubbed_const }
     let(:fits_filename)    { 'fits_0.8.5_pdf.xml' }
     let(:fits_response)    { IO.read(File.join(fixture_path, fits_filename)) }
     let(:file_content)     { 'dummy content' }
     let(:file)             { Hydra::PCDM::File.new { |f| f.content = file_content } }
 
-    before do
+    before(:all) do
```

Comment:
It looks like there is a rubocop issue with this construction so instead how about combining the before and after blocks into an [around block](https://relishapp.com/rspec/rspec-core/v/3-7/docs/hooks/around-hooks) ?
```
around do |example|
  ...
  example.run
  ...
end
```